devel/icu/Portfile

```diff
@@ -8,7 +8,7 @@ set my_name     icu4c
 # whenever the library version number changes. Thanks.
 # To find dependencies, use:
 # port file all | sort -u | xargs grep -El ':icu( |$)' | rev | cut -d / -f 2 | rev | sort -u
-version                     58.2
+version                     61.1
 # Don't update to 59 or later because they require C++11
```

Comment:
Please don't merge yet. As this comment points out, it does not build on systems that don't have C++11 support. Removing this comment line and adding the cxx11 1.1 portgroup lets icu build on 10.6, and I also successfully built harfbuzz-icu using it, but icu-lx failed to build. I haven't investigated further.

Reply:
Thanks for pointing that out. I can't believe I missed this comment.